net/bind/files/bind/named.conf

```diff
@@ -4,8 +4,6 @@ options {
 	directory "/var/cache/bind";
 };
 
-include "/tmp/bind/named.conf.local";
```

Comment:
This comes from 52c4c84700c7b5ecc80099a197d44cfbf4e9cf17. It's probably worth keeping around since it provides an easy mechanism to add additional zone configuration at runtime, while keeping it in `/tmp/` makes it clear that we're not talking about persisting state long-term.

Reply:
Hm...I see. How about putting it in `/var/cache/bind` though? That's kind of the standard location for `bind` files that are temporary in nature (like the dynamic zones that https://github.com/openwrt/packages/commit/52c4c84700c7b5ecc80099a197d44cfbf4e9cf17 refers to)...

On the other hand...how "dynamic" are the zones are we talking about? If it's about the *content* of the zone, then that anyway doesn't end up in `/tmp/bind/named.conf.local` or whatever...it would get written to some other file and referred to from an appropriate zone block (`zone "blah.lan" { type master; file "/tmp/blah.db"; }};`) in the config file....? Or are we really talking about a large bunch of zones that would appear/disappear wholesale from one reboot to the next? Seems a bit....far-fetched?

Reply:
I can't really speak to how any of this is actually being used, as I'm not making use of this feature myself. In terms of the implementation, on OpenWRT either path will be on the same filesystem, so it would be safe. However, it's technically a public interface, so changing the path can potentially break people's configuration.

I'm not aware of any packages within OpenWRT itself that make use of this path, but it's been there for 5 years, so it has had time to get adopted.  If we do change the path, I'd want to keep the commit in `master` and not backport it to any of the existing stable branches (24.10, etc).

Reply:
> I'm not aware of any packages within OpenWRT itself that make use of this path, but it's been there for 5 years, so it has had time to get adopted. If we do change the path, I'd want to keep the commit in `master` and not backport it to any of the existing stable branches (24.10, etc).

Perfectly fine by me... 😄

Reply:
> I can't really speak to how any of this is actually being used, as I'm not making use of this feature myself. In terms of the implementation, on OpenWRT either path will be on the same filesystem, so it would be safe. However, it's technically a public interface, so changing the path can potentially break people's configuration.
> 
> I'm not aware of any packages within OpenWRT itself that make use of this path, but it's been there for 5 years, so it has had time to get adopted. If we do change the path, I'd want to keep the commit in `master` and not backport it to any of the existing stable branches (24.10, etc).

It’s used by ISC-DHCP (and Kea if I can ever get my libubox changes adopted). You should be able to search GitHub for that path.